aws/v2/auth.go

```diff
@@ -0,0 +1,35 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/gruntwork-io/go-commons/errors"
+)
+
+const (
+	DefaultRegion = "us-east-1"
+)
+
+// Options represents all the parameters necessary for setting up authentication credentials with AWS.
+type Options struct {
```

Comment:
Is the idea to expand this in the future with params for, say, IAM role to assume?

Reply:
Yup that's the plan.

Reply:
Roger